spec/models/cancellation_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe Cancellation, model: true do
+  subject { FactoryGirl.build(:cancellation) }
```

Comment:
`FactoryGirl` isn't needed

Reply:
Yep, I know, it's there for consistency.